utils/multiSelectWithoutKeyboard.ts

```diff
@@ -28,13 +28,14 @@ export const multiSelectWithoutKeyboard = <T extends string>(
         target.selected = !target.selected
 
         const value = extractOption(target)
-        const index = arrayToUpdate.value.indexOf(value)
+        if (arrayToUpdate.value) {
+            const index = arrayToUpdate.value.indexOf(value)
+            if (index === -1) {
+                arrayToUpdate.value.push(value)
+                return
+            }
 
-        if (index === -1) {
-            arrayToUpdate.value.push(value)
-            return
+            arrayToUpdate.value.splice(index, 1)
```

Comment:
nice improvement! While I understand it after I read through it, it might be good to add some comments here to help other devs be understand what this function is doing (for example, what is the splice and push doing?